tests/IntegrationTests/Chunks.fs

```diff
@@ -143,4 +143,49 @@ let tests =
              
             Log.Debug("Ended Two parallel chunks-message delivered successfully with short queue")
         }
+
+        testTask "Seek chunk messages and receive correctly" {
+            Log.Debug("Started Seek chunk messages and receive correctly")
+            let client = getClient()
+            let topicName = "public/default/topic-" + Guid.NewGuid().ToString("N")
+            let name = "seekChunkMessages"
+
+            let! (producer : IProducer<byte[]>) =
+                client.NewProducer()
+                    .Topic(topicName)
+                    .ProducerName(name)
+                    .EnableBatching(false)
+                    .EnableChunking(true)
+                    .CompressionType(CompressionType.Snappy)
+                    .CreateAsync() 
+
+            let! (consumer :  IConsumer<byte[]>) =
+                client.NewConsumer()
+                    .Topic(topicName)
+                    .ConsumerName(name)
+                    .StartMessageIdInclusive()
+                    .SubscriptionName("test-subscription")
+                    .SubscribeAsync() 
+
+            let payload = Array.zeroCreate 10_000_000
+            Random().NextBytes(payload)
+            
+            let msgIds = [ for i in 0 .. 9 -> 
+                            producer.NewMessage(payload)
+                            |> producer.SendAsync
+                            |> Async.AwaitTask
+                            |> Async.RunSynchronously]
```

Comment:
Can you please rewrite it similar to [this](https://github.com/fsprojects/pulsar-client-dotnet/blob/develop/tests/IntegrationTests/Chunks.fs#L105-L108)

Reply:
I didn't find a good way to rewrite it. I got some compile errors when I tried it, Could you give me some guidance?